Rakefile

```diff
@@ -6,9 +6,6 @@ require 'rubocop/rake_task'
 
 RuboCop::RakeTask.new do |t|
   t.fail_on_error = false
```

Comment:
I think it's safe to remove this, it was only added for teamcity

Reply:
removed